prep-css-box-model/style.css

```diff
@@ -0,0 +1,17 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+  margin: 20px;
+  padding: 10px;
+  border-style: double;
+  border-color: darkgray;
+  border-width: 3px;
+}
+
+
+/* 👇 YOUR CODE HERE 👇 */
+}
```

Comment:
Looks like you have an extra `}` curly brace chillin here.

Reply:
The code has been updated.